### PR TITLE
Daffa/Config For Schema Contracts

### PR DESCRIPTION
**Pull request by daffahilmyf**
- [X] Config for schema contracts
- [X] Set up Eslint for contracts
- [X] Add Zod for validation

### CODE REVIEW

File: functions/src/index.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import {
   BeforeUserSignedIn,
   CreateUser,
   SyncUser,
-} from "./users/controller";
+} from './users/controller';
 
 dotenv.config();
```
Comment (fadliz) [inline functions/src/index.ts:12]: Duplicate line >:(

**Review by fadliz — COMMENTED** [review]
dupe line on index.ts. others looks good to me!

**Review by Nia-Dalimunthe — APPROVED** [review]
All g.

**Review by fadliz — COMMENTED** [review]
lgtm!

**Pull request merged**